src/templates/blog-post.jsx

```diff
@@ -97,3 +98,19 @@ export const query = graphql`
 `;
 
 export default BlogPostTemplate;
+
+export const Head = ({
+  data: {
+    mdx: { frontmatter },
```

Comment:
```suggestion
    mdx: { frontmatter: {title, summary, ogImage} },
```

Reply:
You can do the same thing for other places, destructuring the `frontmatter` and passing `title`, `description` directly to `SEO` component.